""" TaskAlreadyExists Error
"""


class TaskAlreadyExistsError(Exception):
    """ Exception thrown when trying to create an existing task.
    """
